src/backend/vulkan/src/conv.rs

```diff
@@ -265,10 +265,13 @@ pub fn map_polygon_mode(rm: pso::PolygonMode) -> (vk::PolygonMode, f32) {
     }
 }
 
-pub fn map_cull_face(cf: pso::CullFace) -> vk::CullModeFlags {
-    match cf {
-        pso::CullFace::Front   => vk::CULL_MODE_FRONT_BIT,
-        pso::CullFace::Back    => vk::CULL_MODE_BACK_BIT,
+pub fn map_cull_face(cf: pso::Face) -> vk::CullModeFlags {
+    if cf.contains(pso::Face::FRONT) {
```

Comment:
same here - needs to handle `FRONT | BACK`

Reply:
Sure 👍 I was trying to avoid changing the behavior too much, but I think we should fix this for #2067